templates/repo/header.tmpl

```diff
@@ -6,15 +6,51 @@
 				{{if .RelAvatarLink}}
 					<img class="ui avatar image" src="{{.RelAvatarLink}}">
 				{{else}}
-					{{template "repo/header_icon" .}}
+					{{if .IsTemplate}}
+						{{if .IsPrivate}}
+							{{svg "octicon-repo-template-private" 32}}
+						{{else}}
+							{{svg "octicon-repo-template" 32}}
+						{{end}}
+					{{else}}
+						{{if .IsPrivate}}
+							{{svg "octicon-lock" 32}}
+						{{else if and (not .IsMirror) (not .IsFork) ($.Owner)}}
+							{{if .Owner.Visibility.IsPrivate}}
+								{{svg "octicon-internal-repo" 32}}
+							{{else}}
+								{{svg "octicon-repo" 32}}
+							{{end}}
+						{{else if .IsMirror}}
+							{{svg "octicon-repo-clone" 32}}
+						{{else if .IsFork}}
+							{{svg "octicon-repo-forked" 32}}
+						{{else}}
+							{{svg "octicon-repo" 32}}
+						{{end}}
+					{{end}}
```

Comment:
Would've still kept the `repo/header_icon` file. Even if it's not duplicated, it's still nice to have file-level abstraction sometimes.